### PR TITLE
add a configuration for the window title's tied tag

### LINKED ISSUE #2254: Custom window title
I would like my window title to be "title - artist", rather than "title - people"
I have not found a way to accomplish this.

### PR DESCRIPTION
**Pull request by ptitjes**
Fixes #2254.

### CODE REVIEW

**Comment by ptitjes** [comment]
@lazka Made an advanced preference of it.

File: quodlibet/quodlibet/qltk/quodlibetwindow.py [hunk 1]
```diff
@@ -1255,11 +1255,15 @@ def __song_changed(self, library, songs, player):
         if player.info in songs:
             self.__update_title(player)
 
+    def update_title(self):
```
Comment (lazka) [inline quodlibet/quodlibet/qltk/quodlibetwindow.py]: unused?

Comment (ptitjes) [inline quodlibet/quodlibet/qltk/quodlibetwindow.py]: Yep, preference cleaning not done completely...

**Comment by lazka** [comment]
Thanks!

**Comment by ptitjes** [comment]
Thank you.

**Pull request merged**